tensorflow_encrypted/tensor/int100.py

```diff
@@ -115,7 +115,7 @@ def eval(self, sess: tf.Session, feed_dict: Dict[Any, Any]={}, tag: Optional[str
         evaluated_backing = run(sess, self.backing, feed_dict=feed_dict, tag=tag)
         return Int100Tensor.from_decomposed(evaluated_backing)
 
-    def to_int32(self) -> Union[tf.Tensor, np.ndarray]:
+    def to_native(self) -> Union[tf.Tensor, np.ndarray]:
```

Comment:
not sure what's best here, but you might be right that it should be `native`. my issue is that the code implicitly chops off anything outside the int32 range, and is converting numbers into that dtype. 

what was the need for renaming it?

Reply:
All the tensors need to share a similar function since the `to_int32` function is being used when decoding the numbers in pond. Didn't want it to be called `to_int32` in the `Int64Tensor`.

Reply:
yeah make sense. there is something underlying here we haven't solved yet but it can be pushed till later when we refactor/figure out what to do with the different tensors